src/scikit_hep_repo_review/processor.py

```diff
@@ -105,3 +105,14 @@ def process(package: Traversable) -> dict[str, list[Result]]:
         results_dict[family] = result_list
 
     return results_dict
+
+
+def results_dict_to_json(results_dict: dict) -> str:
```

Comment:
```suggestion
def results_dict_to_json(results_dict: dict[str, list[Result]) -> str:
```

Though I'd probably return the object and put the json.dumps in the CLI. I can do this soon, thanks!

Reply:
Personally I would want to access the JSON stuff from API, not CLI. But however you choose to do this, I'll make it with on my end. Thanks again!

Reply:
Yes, but it's easy to wrap with `json.dumps`, and then you can add indent, etc. if you want.

Reply:
Oh yeah, I forgot about the indent. That would be nice.

Reply:
Your example actually already wrapped it in json.dumps. So the example of usage stays the same. ;)